package/yast2-packager.changes

```diff
@@ -1,3 +1,11 @@
+-------------------------------------------------------------------
+Wed Apr  6 14:00:54 UTC 2022 - Stefan Hundhammer <shundhammer@suse.com>
+
+- Show package downloads in the global progress bar during package
+  installation (bsc#1195608)
+  PR: https://github.com/yast/yast-packager/pull/609
```

Comment:
Was the PR URL intentionally added?

Reply:
Yes, so it's easier to get to the PR with more explanations. I started doing that some weeks ago after I had to hunt down a PR.